react/src/pages/analysis/Analysis.tsx

```diff
@@ -224,8 +227,7 @@ export default function Analysis() {
                     options={{
                         pageSize: 10,
                         filtering: true,
-                        search: false,
-                        padding: 'dense'
```

Comment:
I think we want to keep `padding: dense,` beacause we are able to see more rows with this parameter. For the participant table on dashboard, I added `copyIcon: {
        marginLeft: theme.spacing(1),
    }`to style the left most action button so the table looks more symmetrical with the extra margin. Maybe you can also add this to the visibility icon.